deploy-local.sh

```diff
@@ -18,10 +18,14 @@ BSC_TESTNET_PRIVATE_KEY=acbaa26******************************a88367d9
 export PATH=$PATH:/usr/local/go/bin:/usr/local/go/bin:/root/go/bin
 echo '0. stop old database/redis and docker run new database/redis'
 pm2 delete all
-docker kill $(docker ps -q)
-docker rm $(docker ps -a -q)
-docker run -d --name zkbnbredis -p 6379:6379 redis
-docker run --name postgres -p 5432:5432 -e PGDATA=/var/lib/postgresql/pgdata  -e POSTGRES_PASSWORD=ZkBNB@123 -e POSTGRES_USER=postgres -e POSTGRES_DB=zkbnb -d postgres
+docker kill $(docker ps -a |grep zkbnb|awk '{print $1}')
+docker rm $(docker ps -a |grep zkbnb|awk '{print $1}')
```

Comment:
There will an error ```Error response from daemon: Cannot kill container: 254eb3639889: Container 254eb3639889ec805c695365a9993c840779eddb854c48e0f57392ff5e207f59 is not running
``` when the images does not exist or not running

```suggestion remove docker kill and add -f  to docker rm
docker rm -f $(docker ps -a |grep zkbnb|awk '{print $1}')
```